apps/react-world/src/components/login/LoginForm.tsx

```diff
@@ -73,18 +76,36 @@ const LoginForm = (props: LoginFormProps) => {
           <LoginErrorMessage>{errors.password.message}</LoginErrorMessage>
         )}
       </fieldset>
-      <button type="submit" className="btn btn-lg btn-primary pull-xs-right">
-        Sign in
-      </button>
+      {loginError && (
+        <LoginErrorMessage>
+          {Object.entries(loginError)
+            .map(([key, value]) => `${key} ${value[0]}`)
+            .join('. ')}
+        </LoginErrorMessage>
+      )}
+      <LoginButton disabled={shouldLoginButtonDisabled}>
+        {buttonText}
+      </LoginButton>
     </form>
   );
 };
 
-const LoginErrorMessage = styled.p`
-  color: red;
-  margin-top: 5px;
-  margin-left: 5px;
-  font-size: 14px;
-`;
+interface LoginButtonProps {
+  disabled: boolean;
+  children?: ReactNode;
+}
```

Comment:
```suggestion
interface LoginButtonProps extends PropsWithChildren<{disabled: boolean}> {}
```
확장 가능성을 주지 않기 위해 Button 속성을 주지 않은 것일까요?
그렇다면 children은 기존에 있는 타입을 활용해주는 것이 소통하기 편하답니다!

Reply:
@InSeong-So Button 속성으로 구현하는 방향이 있었군요..! 아래처럼 구현했습니다!

```tsx
interface LoginButtonProps
  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  children?: ReactNode;
}
const LoginButton = (props: LoginButtonProps) => {
  const { disabled = false, children, ...restProps } = props;

  return (
    <StyledLoginButton
      type="submit"
      className={`btn btn-lg pull-xs-right`}
      disabled={disabled}
      {...restProps}
    >
      {children}
    </StyledLoginButton>
  );
};
```